mailyak.go

```diff
@@ -47,6 +48,7 @@ func New(host string, auth smtp.Auth) *MailYak {
 		auth:           auth,
 		trimRegex:      regexp.MustCompile("\r?\n"),
 		writeBccHeader: false,
+		date:           currentDateTime,
```

Comment:
This should be set to `time.Now().Format(time.RFC1123Z)` for each call, rather than as a global.